### ISSUE TITLE
Armor models do not load after update 1.0

### ISSUE THREAD

**Issue by ClassicsWoTWorkshop** [smellyriver/tankinspector#18]
Armor models don't load since update 1.0 (only the visual model works, but the penetration indicator does not). The collision model format was changed to .havok with the game update, which seems to be the cause of this issue.

**Comment by adri1wcrow** [comment]
Yes, it changed to the havok and I'm not sure how to fix it. @hillin

**Comment by Edrard** [comment]
Like I know, its no fix for it. After Havoc was sold to MS, they stopped distribute free SDK.

**Comment by hillin** [comment]
I've no idea about the havok models, but this may involve additional reverse engineering and a thorough refactor to the armor model system, which is very likely to be a big project. The original RE works are done by our partner @zooedward , but he has everything to keep him busy enough, so I guess we are unable to fix this in the near future, unfortunately.

**Comment by ghost** [comment]
https://github.com/blueskythlikesclouds/TagTools
Here is a project that parses havok files. It is a little outdated, but the basic principles remained the same.

**Comment by Edrard** [comment]
adri1wcrow or hillin, please, contact MasterModeley directly or me over email (demonand@gmail.com) and I will help you contact with MasterModeley. He have info, about havok form and will to help.

**Comment by adri1wcrow** [comment]
I don't know much about the model and I'm busy with my family recently. So I will work on the software update first.

I left the operator company of WOT in China more than half a year.
The Chinese server still work on the 9.22 version and don't have the update plan now.
So I can only spend a little time on maintain the tankinspector. Sorry.
